script.js

```diff
@@ -0,0 +1,398 @@
+    pvpButton.disabled = true
+    nameForm.addEventListener('submit', addNames)
+
+    startButton.addEventListener('click', () => {
+        submit.disabled = true
+        timerStatus = "stopped"
+        document.getElementById("time").innerText = "STARTING GAME COUNTER...."
+        startStop();
+        game1.innerText = ''
+        game1.style.backgroundColor = "white"
+        game2.innerText = ''
+        game2.style.backgroundColor = "white"
+        game3.innerText = ''
+        game3.style.backgroundColor = "white"
+        game4.innerText = ''
+        game4.style.backgroundColor = "white"
+        game5.innerText = ''
+        game5.style.backgroundColor = "white"
+        game6.innerText = ''
+        game6.style.backgroundColor = "white"
+        game7.innerText = ''
+        game7.style.backgroundColor = "white"
+        game8.innerText = ''
+        game8.style.backgroundColor = "white"
+        game9.innerText = ''
+        game9.style.backgroundColor = "white"
+        startButton.disabled = true;
+        status.innerText = nameOne.value + '\'s turn!'
+        player = 'x'
+        gameOver = false
+
+        clickedCell.forEach(function (clickedCellArrayItem) {
+            clickedCellArrayItem.addEventListener("click", turnsPVP)
+        })
+    })
+}
+// The blow function handles player vs player turns
+function turnsPVP(evt) {
+    if (player === 'x' && gameOver === false) {
+        evt.target.textContent = 'x'  //an x appears in that cell
+        player = "o"  // and the turn ends
+        status.innerText = nameTwo.value + '\'s turn!' // and the current player changes from x to o
+        evt.target.removeEventListener('click', turnsPVP) // this keeps the player from selecting a filled cell
+        evt.target.addEventListener('click', emptyAlert)
+        determineWin();
+    } else if (player === 'o' && gameOver === false) {
+        evt.target.textContent = 'o'  //an o appears in that cell
+        player = 'x'   // and the turn ends
+        status.innerText = nameOne.value + '\'s turn!' // and the current player changes from o to x
+        evt.target.removeEventListener('click', turnsPVP) // this keeps the player from selecting a filled cell
+        evt.target.addEventListener('click', emptyAlert)
+        determineWin();
+    }
+}
+ 
+// ****************** General Functions **************** //
+// The below function creates an alert when a player tries to select a full cell
+function emptyAlert(evt) {
+    alert('please select an empty cell')
+}
+
+// The below function checks for win conditions and the program's response
+function determineWin() {
+    if (game1.textContent === 'x' && game2.textContent === 'x' && game3.textContent === 'x') {
+        game1.style.backgroundColor = "red"
+        game2.style.backgroundColor = "red"
+        game3.style.backgroundColor = "red"
+        status.innerText = `Game Over! ${nameOne.value} wins!`
+        gameOver = true
+        resetTime();
+        reset();
```

Comment:
You could rephrase your argument in these to cut the number of logic statements in half. As of right now you have one written for every win condition for both X and O. If we change these so that it's just checking if the text contents are the same, rather then comparing them to something, it would remove 1/2 of the logic blocks.